tket/src/Utils/MatrixAnalysis.cpp

```diff
@@ -570,41 +452,40 @@ get_information_content(const Eigen::Matrix4cd &X) {
       Eigen::kroneckerProduct(PauliX, Eigen::Matrix2cd::Identity());
   const Eigen::Matrix4cd s_ix =
       Eigen::kroneckerProduct(Eigen::Matrix2cd::Identity(), PauliX);
-  if (k(0) > PI / 2) {
-    k(0) -= 1.5 * PI;
-    K1 *= -i_ * s_xx;
+  if (k(0) > 1.) {
+    k(0) -= 3.;
+    K1 *= i_ * s_xx;
```

Comment:
Why did the sign of `K1` change here (and in a few other places below)?

Reply:
Because the new `k(0)` has an additional sign flip compared to the old `k(0)` (the change of sign of the matrix `P` higher up (line 27)). Hence projecting `k(0)` into the Weyl chamber requires a "move" in the opposite direction.

I spent a lot of time getting these operations right, I'm fairly confident they are correct now.

Reply:
Sorry, I didn't explain myself properly: "I'm confident they are correct _now_" refers to earlier as I was developing this. It took a fair bit of algebra + testing to make sure every case was covered properly. This is what all the tests on lines 1773 in `test_Synthesis.cpp` and below are testing. If you look at the coverage, you will indeed see that all the cases are covered.